discover/portal/articles/210-setting-up/02-instance-settings/02-instance-settings.markdown

```diff
@@ -0,0 +1,217 @@
+# Instance Settings [](id=instance-settings)
+
+Virtual instance settings are configured in the Instance Settings section of the
+Control Panel. 
+
+Instance Settings has four sections: *Configuration*, *Identification*,
+*Social*, and *Miscellaneous*. This article covers the first and largest
+section, Configuration. The remaining sections are covered in the next article.
+
+![Figure 1: The configuration options in Instance Settings are divided into four categories.](../../../images/instance-settings-navigation.png)
+
+The Configuration section of In stance Settings is divided into seven
```

Comment:
Wrong space. "In stance" should be "Instance"

Reply:
Sorry, typos shouldn't get to the SME reviewer. I just don't have as much time as I'd like to review my own drafts before sending them right now. Thanks for catching these things.